Paper.tex

```diff
@@ -203,6 +203,7 @@ \subsection{Which History?}
 $F_{\mathrm{Arrow Glacier}}$     & 13773000 \\
 $F_{\mathrm{Gray Glacier}}$      & 15050000 \\
 $F_{\mathrm{Paris}}$             & 15537394 \\
+$F_{\mathrm{Shanghai}}$          & 17034870 \\
```

Comment:
Out of scope of this PR, was thinking if going forward there should be another column besides `Block Number`, e.g. `Timestamp` or whatever will be the canonical trigger.

Reply:
That's a good idea imo!